pkg/plugin/standalone/v1/destination.go

```diff
@@ -18,13 +18,13 @@ import (
 	"context"
 	"io"
 
+	connectorv1 "github.com/conduitio/conduit-connector-protocol/proto/gen/connector/v1"
```

Comment:
Hmm, so if the alternative is putting the code in `conduit-connector-protocol/connector/v1` I'm not a fan, we have other manually written Go code in that project so I like that the proto code is separated from it in a separate folder.

That said, I'm not married to `gen`, for all I care we can generate the code directly into the `proto` folder so the path would be `conduit-connector-protocol/proto/connector/v1`, the same folder that contains the source proto file. I've seen other projects generate Go code alongside the proto files, I'm ok with this if you want to change it.

Reply:
There were two reasons I put in `proto/gen` initially. One is to be consistent with what we have in Conduit, but I don't think consistency actually matters there (internal vs external usage). The other one was to be able to easily clean it up, but given that Buf will overwrite it, it's probably the same.

Reply:
I see the appeal of deleting the code by deleting the entire folder. It's still doable by deleting all `.go` files in `proto`, not as easy but still easy enough I think. That said, I don't have strong feelings if we should use `proto` or `proto/gen`, I see benefits in both approaches and neither will make a huge difference. All I ask for is to change it in both projects in case we change it, I do like consistent codebases 🙂

Reply:
Cool, thanks for your thoughts! I'm slightly inclined towards just `proto`, given that `proto/gen` didn't appear as pretty in conduit-connector-protocol import paths.:) I'll make them consistent.